_quarto.yml

```
@@ -7,7 +7,7 @@ project:
 
 website:
   title: "R-multiverse"
-  description: "A community-curated collection of R package releases, powered by R-universe."
+  description: "A dual package repository system, powered by R-universe"
```

```suggestion
  description: "A dual package repository system, powered by R-universe and curated by the community"
```

or something else allowing to state it's community-curated

---

"A community-curated package repository system, powered by R-universe"

The fact it's 'dual' is not integral to its purpose. In fact we also have a 3rd staging repo as you commented below....